### PR TITLE
Activate jenkins PR jobs for kogito-tooling

### PR DESCRIPTION
**Pull request by jomarko**
This change is needed due to monorepo-migration:
    - https://github.com/kiegroup/kogito-tooling/pull/729

### CODE REVIEW

File: dsl/seed/config/branch.yaml [hunk 1]
```diff
@@ -41,9 +41,9 @@ repositories:
 # Main branch configuration
 git:
   author:
-    name: kiegroup
-    credentials_id: kie-ci
-    token_credentials_id: kie-ci2-token
+    name: jomarko
+    credentials_id: jomarko
+    token_credentials_id: jomarko-gh-token
   bot_author:
```
Comment (mareknovotny) [inline dsl/seed/config/branch.yaml]: this needs to be changed from personal to kiegroup ones

Comment (jomarko) [reply dsl/seed/config/branch.yaml]: @mareknovotny yes, once debug of my new job is finished, I will revert the second commit of this PR

**Comment by jomarko** [comment]
@mareknovotny @radtriste FYI

Jenkins job passed 
- https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/blue/organizations/jenkins/custom%2Fjomarko%2FPR-MONOREPO-MIGRATION%2Fpullrequest%2Fkogito-tooling.tests.kogito-tooling-stunner-editors/detail/kogito-tooling.tests.kogito-tooling-stunner-editors/14/pipeline

with results available as
- https://sonarcloud.io/summary/new_code?id=org.kie.kogito%3Akogito-tooling&pullRequest=740

I opened PR as
- https://github.com/tiagobento/kogito-tooling/pull/41/files

I will revert the changes I used to be able to debug new job.

**Comment by radtriste** [comment]
@jomarko should we merge this one ?

**Comment by jomarko** [comment]
We can wait until kiegroup/kogito-tooling#729 is in @radtriste  I think

**Comment by jomarko** [comment]
@radtriste please merge this pr, also please let me know if I need somehow start job re-generation

**Comment by radtriste** [comment]
@jomarko job generation should be automatic

**Pull request merged**